one_fm/patches.txt

```diff
@@ -120,8 +119,8 @@ one_fm.patches.v15_0.update_operation_post_start_and_end_date
 one_fm.patches.v15_0.add_custom_process_field_property_setter
 one_fm.patches.v15_0.add_update_custom_field #ToDo field
 one_fm.patches.v15_0.add_update_property_setter #2025-05-20
-one_fm.patches.v15_0.add_update_assignment_rule #2025-07-2
-one_fm.patches.v15_0.add_update_workflow #2025-06-04
+one_fm.patches.v15_0.add_update_assignment_rule #2025-08-17
+one_fm.patches.v15_0.add_update_workflow #2025-08-17
```

Comment:
Review the code standard and create separate patches for update the specific assignment rule and workflow instead of updating all the workflow/assignment rule